### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by DungNH41**
Cap nhat

### CODE REVIEW

File: New folder/abc.txt [hunk 1]
```diff
@@ -2,14 +2,3 @@
 2
 3
 4
```
Comment (DungNH41) [inline New folder/abc.txt]: ahsfdgjhkagfjshkgda

File: New folder/abc.txt [hunk 2]
```diff
@@ -1,15 +1,4 @@
 1
 2
 3
-4
 5
```
Comment (DungNH41) [inline New folder/abc.txt]: comment

File: New folder/a [hunk 3]
```diff
@@ -0,0 +1 @@
+1
```
Comment (DungNH41) [inline New folder/a]: chinh lai 2

File: New folder/a [hunk 4]
```diff
@@ -0,0 +1 @@
+2
```
Comment (DungNH41) [inline New folder/a]: sua lai 3

Comment (HuanNB6) [inline New folder/a]: Comment @DungNH41

**Review by HuanNB6 — APPROVED** [review]

**Pull request closed without merging**